otherlibs/win32graph/draw.c

```diff
@@ -304,13 +304,22 @@ static value gr_draw_or_fill_arc(value vx, value vy, value vrx, value vry,
         return Val_unit;
 }
 
+/*
+ * The following function, caml_gr_show_bitmap, is broken:
+ * 1. AfficheBitmap which it calls is not exported (defined in dib.c)
+ * 2. The types of passed arguments does not match the expected ones:
+ *   void AfficheBitmap(char *filename,HDC hDC,int x,int y)
+ * 3. AfficheBitmap is called twice, which is probably a mistake
+ * Since this function is not used anywhere, it is disabled 
+ */
+#if 0
```

Comment:
I'd remove this completely. Indeed, I don't know how this could be working and I don't like commented-out code.

Moreover you can remove AfficheBitmap(). And ChargerBitmap() because only AfficheBitmap() uses it. And it probably expands to most of dib.c if not all of it.